docroot/modules/custom/govcms_draft_content_access/src/Plugin/Block/GovcmsDraftContentAccessBlock.php

```diff
@@ -200,33 +200,55 @@ public function build() {
           ],
         ]);
 
-        $rows[$row->id] = [
-          'link' => \Drupal::l($this->t('Copy'), $link_html),
-          'plain_link' => \Drupal::l($this->t('Right click and copy link'), $link_plain),
+        $rows[$row->id]['data'] = [
+          'link' => [
+            'data' => [
+              '#type' => 'link',
+              '#title' => $this->t('Copy'),
+              '#url' => $link_html,
+            ],
+          ],
+          'plain_link' => [
+            'data' => [
+              '#type' => 'link',
+              '#url' => $link_plain,
+              '#title' => $this->t('Right click and copy link'),
+            ],
+          ],
           'expired' => $date_format_interval,
-          'operation' => \Drupal::l($this->t('Revoke link'), $revoke_url),
+          'operation' => [
+            'data' => [
+              '#type' => 'link',
+              '#url' => $revoke_url,
+              '#title' => $this->t('Revoke link'),
+            ],
+          ],
         ];
       }
 
-      $generate_url = Url::fromRoute('govcms_draft_content_access.generatelink', [
-        'uid' => $uid,
-        'destination' => str_replace("/", "-", $destination),
-      ], [
-        'attributes' => [
-          'class' => [
-            'btn',
-            'btn-success',
+      $generate_url = [
+        '#type' => 'link',
+        '#url' => Url::fromRoute('govcms_draft_content_access.generatelink', [
+          'uid' => $uid,
+          'destination' => str_replace("/", "-", $destination),
+        ], [
+          'attributes' => [
+            'class' => [
+              'btn',
+              'btn-success',
+            ],
           ],
-        ],
-      ]);
+        ]),
+        '#title' => $this->t('Generate link'),
+      ];
 
       $build = [
-        '#prefix' => $this->t('<p>List of existing access link to this content. Click revoke to remove existing/expired link.</p>'),
+        '#prefix' => '<p>' . $this->t('List of existing access link to this content. Click revoke to remove existing/expired link.') . '</p>',
         '#type' => 'table',
         '#header' => $header,
         '#rows' => $rows,
         '#empty' => $this->t('No autologin link found for this node.'),
-        '#suffix' => \Drupal::l($this->t('Generate link'), $generate_url),
+        '#suffix' => render($generate_url),
```

Comment:
Can't use a render array here because of the way the template renders the suffix.